docker/minimal/dalc-docker-compose.yml

```diff
@@ -3,26 +3,26 @@ version: '3'
 services:
   dalc0:
     container_name: dalc0
-    image: "ghcr.io/celestiaorg/dalc:sha-b46432b"
+    image: "ghcr.io/celestiaorg/celestia-node:sha-042ebe1"
     environment:
-      - NODE_TYPE=light
+      - NODE_TYPE=bridge
       - MY_VAL_ADDRESS=celes1dm7fp7uwrr5y68v48gg8jhd2cku7vt7f4l8w0s
-      - CELESTIA_PRIVATE_GENESIS=942D98EA04EE31F523D5B5BF5B5E801CEB5BC4AD060BBF5B656E6C30A0EDCE1B
+      - CELESTIA_PRIVATE_GENESIS=02EED4BCFAF0CC8A3D1100D588E571FBF44219488F01AB48870BC9015A9F105A
     command: [
-            "celestia", "light",
+            "celestia", "bridge", "start",
```

Comment:
the issue with init is in our entrypoint.sh I suppose 🧐 
https://github.com/celestiaorg/celestia-node/blob/main/docker/entrypoint.sh

```bash
if [ "$1" = 'celestia' ]; then
    ./celestia "${NODE_TYPE}" --node.store /celestia-"${NODE_TYPE}" init

    exec ./"$@" "--"
fi
```

Reply:
This is fixed in https://github.com/celestiaorg/celestia-node/pull/845